### PR TITLE
Update simplify

### PR DESCRIPTION
**Pull request by thepiwo**

### CODE REVIEW

**Review by mmpetarpeshev — APPROVED** [review]
LGTM. I tested the build and deployment. Working on the pipelines now.

File: aepp/src/utils/aeternityNetwork.js [hunk 1]
```diff
@@ -86,31 +85,11 @@ aeternity.hasActiveWallet = () => {
...
-  return aeternity.networkId === 'ae_mainnet';
-};
-
-
 /**
  * Initializes the aeternity sdk to be imported in other occasions
  * @returns {Promise<boolean>}
  */
 aeternity.initClient = async () => {
-  if (process && process.env && process.env.PRIVATE_KEY && process.env.PUBLIC_KEY) {
```
Comment (kenodressel) [inline aepp/src/utils/aeternityNetwork.js:103]: i think this is relevant for the tests

**Review by kenodressel — APPROVED** [review]

File: aepp/src/utils/aeternityNetwork.js (hunk 1 above)
Comment (thepiwo) [reply aepp/src/utils/aeternityNetwork.js:103]: are they still working at all, at least I couldn't see those envs defined anywhere.

Comment (kenodressel) [reply aepp/src/utils/aeternityNetwork.js:103]: does not look like we ever wrote any :shrug:

**Pull request merged**